app/views/partials/droits-eligibles-list.html

```diff
@@ -1,96 +1,92 @@
-<uib-accordion close-others="false">
-  <uib-accordion-group is-open="droit.open"
-     class="result {{ getAccordionClass(droit) }}"
-     ng-repeat="(droitId, droit) in list|orderObjectBy:'label'">
-    <uib-accordion-heading>
-      <div class="row">
-        <div class="col-xs-2 col-sm-1">
-          <img class="droit-logo"
-               ng-src="/img/{{ droit.imgSrc }}"
-               alt="">
+<table class="table table-result">
+  <tbody ng-repeat="(droitId, droit) in list|orderObjectBy:'label'">
+    <tr class="result {{ getAccordionClass(droit) }}" ng-click="droit.open = !droit.open" ng-class="{'opened': openend === droitId}">
+      <th>
+        <img class="droit-logo"
+           ng-src="/img/{{ droit.imgSrc }}"
+           alt="Caisse d'allocations familiales">
+      </th>
+      <th>{{ droit.label }}</th>
+      <th>
+        <div class="montant">
+          <span ng-if="isNumber(droit.montant)">{{ round(droit) }}&nbsp;{{ droit.unit || '€' }}</span>
+          <span ng-if="isString(droit.montant)" class="montant-inconnu">?</span>
         </div>
-
-        <div class="col-xs-6 col-sm-8 description">
-          <span>{{ droit.label }}<span>
+        <div class="montant-detail">
+          <span ng-if="isNumber(droit.montant)">{{ droit.legend || (droit.isMontantAnnuel ? '/ an' : '/ mois') }}</span>
         </div>
-
-        <div class="col-xs-3 col-sm-2">
-          <div class="montant">
-            <span ng-if="isNumber(droit.montant)">{{ round(droit) }}&nbsp;{{ droit.unit || '€' }}</span>
-            <span ng-if="isString(droit.montant)" class="montant-inconnu">?</span>
-            <i class="fa fa-warning text-warning" ng-if="shouldDisplayYM2Warning(droit)"></i>
+      </th>
+      <th>
+        <i class="fa fa-warning text-warning" ng-if="shouldDisplayYM2Warning(droit)"></i>
+        <span class="screenReadersOnly">Cette aide se base sur vos ressources de l'année {{ yearMoins2 }}</span>
+      </th>
+      <th class="chevron">
+        <button class="btn btn-sm btn-link">
+          <i class="fa" ng-class="{'fa-chevron-down': droit.open, 'fa-chevron-right': !droit.open}" aria-hidden="true"></i>
```

Comment:
You also need to add `role="presentation"` (see https://github.com/sgmap/mes-aides-ui/commit/f190493d248fe7fb6153b08b01cacb243a4a83d3).